mods/d2k/sequences/structures.yaml

```diff
@@ -35,9 +35,9 @@ guntower:
 		Facings: -32
 		Offset: -24,16
 	muzzle: DATA.R8
-		Frames: 3839, 3839, 3840, 3840, 3841, 3841, 3842, 3842, 3843, 3843, 3844, 3844, 3845, 3845, 3846, 3846, 3847, 3847, 3848, 3848, 3849, 3849, 3850, 3850, 3851, 3851, 3852, 3852, 3853, 3853, 3854, 3854, 3855, 3855, 3856, 3856, 3857, 3857, 3858, 3858, 3859, 3859, 3860, 3860, 3861, 3861, 3862, 3862, 3863, 3863, 3864, 3864, 3865, 3865, 3866, 3866, 3867, 3867, 3868, 3868, 3869, 3869, 3870, 3870
+		Start: 3839
+		Tick: 60
 		Facings: -32
-		Length: 2
```

Comment:
Until @pchote fixed that bug in #7483, the first frame of a non-looping animation would never show (even with exaggerated ```Tick:``` values).

To work around that, I duplicated the muzzle frames way back because the 2nd frame of an animation was shown normally (99% of the time, at least). Obviously a very ugly hack, which I wanted to get rid of asap.

Now that the bug is fixed, I thought it was time to get rid of that ugly work-around.
In case you're wondering, 1-frame animations still need a ```Tick:``` value higher than 40 to show up, but according to pchote that's a different, high-level issue with ```WithMuzzleFlash```, ```WithDeathAnimation``` etc. since they call ```anim.Tick``` unconditionally in their first tick and don't do a ```if (!firstTick)``` check (that's what I remember, ask him if you want more details).